tracer/test/Datadog.Trace.TestHelpers/TestHelper.cs

```diff
@@ -661,6 +675,86 @@ protected void EnableTelemetry(bool enabled = true, int? standaloneAgentPort = n
             Assert.Equal(expectedServiceVersion, span.Tags.GetValueOrDefault(Tags.Version));
         }
 
+        protected void SetupRcmConfiguration(string path = null)
+        {
+            if (path == null)
+            {
+                path = Path.GetTempFileName();
+                File.Delete(path);
+                Directory.CreateDirectory(path);
+                path = Path.Combine(path, "rcm_config.json");
+                if (File.Exists(path)) { File.Delete(path); }
+                _deleteRcmFile = true;
+            }
+            else
+            {
+                _deleteRcmFile = false;
+            }
+
+            SetEnvironmentVariable(ConfigurationKeys.Rcm.FilePath, path);
+        }
+
+        protected void CleanupRcmConfiguration()
+        {
+            if (EnvironmentHelper.CustomEnvironmentVariables.TryGetValue(ConfigurationKeys.Rcm.FilePath, out var rcmConfigPath))
+            {
+                if (File.Exists(rcmConfigPath) && _deleteRcmFile) { File.Delete(rcmConfigPath); }
+                EnvironmentHelper.CustomEnvironmentVariables.Remove(ConfigurationKeys.Rcm.FilePath);
+            }
+        }
+
+        protected void SetRcmConfiguration(IEnumerable<(object Config, string Id)> configurations, string productName)
```

Comment:
I think the name of method is misleading.
Method writes RCM to a file, so probably use `WriteRcmToFile`?

Reply:
I inherited this, but we can idscuss it. I find SeRcmConfiguration is understandable, but I'm open to change it.